homeassistant/components/tuya/sensor.py

```diff
@@ -840,6 +840,91 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
             device_class=SensorDeviceClass.POWER,
             state_class=SensorStateClass.MEASUREMENT,
         ),
+        # Pool heat pump - power consumption
+        TuyaSensorEntityDescription(
+            key=DPCode.POWER,
+            translation_key="power_consumption",
```

Comment:
I suggest to keep the comment "inside" the description rather than above;
(This applies to all new descriptions)
```suggestion
        TuyaSensorEntityDescription(
            # Pool heat pump - power consumption
            key=DPCode.POWER,
            translation_key="power_consumption",
```